content/includes/footer.html

```diff
@@ -145,6 +145,7 @@ <h3>Get help from Veterans Crisis Line</h3>
 })();
 </script>
 {% endif %}
+<script type="text/javascript" src="https://resources.digital-cloud-gov.voice.medallia.com/wdcgov/2/onsite/embed.js" async></script>
```

Comment:
@wyattwalter @patrickvinograd It looks like we'll need to whitelist this domain if it isn't already. This script has to be served from that domain.